pkg/querier/queryrange/split_by_interval.go

```diff
@@ -219,3 +238,20 @@ func splitByTime(r *LokiRequest, interval time.Duration) []queryrange.Request {
 	}
 	return reqs
 }
+
+func splitSeriesByTime(r *LokiSeriesRequest, interval time.Duration) []queryrange.Request {
```

Comment:
This can be consollidated by having the `splitByTime` function accept a `queryrange.Request` instead of a `*LokiRequest` and then using the `WithStartEnd` method of that interface.